### PR TITLE
Elwayman02 upgrade packages

### PR DESCRIPTION
**Pull request by sglanzer**
Slightly updated PR from the ember-cli upgrade

### CODE REVIEW

**Comment by elwayman02** [comment]
Supersedes #88

File: .idea/vcs.xml [hunk 1]
```diff
@@ -2,5 +2,6 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/node_modules/ember-cli" vcs="Git" />
```
Comment (elwayman02) [inline .idea/vcs.xml]: these .idea files should be removed :)

File: tests/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Dummy Tests</title>
+    <title>Ember-CLI-Blanket Demo</title>
```
Comment (elwayman02) [inline tests/index.html]: We should probably revert this, was my bad.  I got confused as to which index.html I was looking at.

File: .idea/vcs.xml (hunk 1 above)
Comment (sglanzer) [inline .idea/vcs.xml]: Yeah, I was going to wait until post-merge to remove that directory

**Comment by sglanzer** [comment]
Merging - will close #88 - I'm not going to publish 0.6.0 until I get a chance to clean up a few more things though.  Maybe a few days, not too long.

**Pull request merged**